beacon-chain/rpc/testutil/mock_state_fetcher.go

```diff
@@ -8,10 +8,16 @@ import (
 
 // MockFetcher is a fake implementation of statefetcher.Fetcher.
 type MockFetcher struct {
-	BeaconState iface.BeaconState
+	BeaconState     iface.BeaconState
+	BeaconStateRoot []byte
 }
 
 // State --
 func (m *MockFetcher) State(context.Context, []byte) (iface.BeaconState, error) {
 	return m.BeaconState, nil
 }
+
+// State --
```

Comment:
```suggestion
// StateRoot --
```

Not that it matters, but yeah... :grin:

Reply:
:relieved: